packages/react-component-library/src/components/Button/partials/StyledButton.tsx

```diff
@@ -6,172 +6,134 @@ import { BUTTON_ICON_POSITION, BUTTON_VARIANT } from '../constants'
 import { ButtonIconPositionType, ButtonVariantType } from '../Button'
 import { ComponentSizeType, COMPONENT_SIZE } from '../../Forms'
 
+const { color, spacing, fontSize, shadow } = selectors
+
+const DROP_SHADOW = `0 1px 6px ${rgba(0, 0, 0, 0.3)}`
```

Comment:
Could make this a shadow token if we wanted (it isn't used anywhere else though, and it'd have to go at the end of the list).